creator-node/src/index.ts

```diff
@@ -89,15 +80,61 @@ const startApp = async () => {
     throw new Error('Invalid delegatePrivateKey/delegateOwnerWallet pair')
   }
 
+  // Fail if Trusted Notifier isn't configured properly
   const trustedNotifierEnabled = !!config.get('trustedNotifierID')
   const nodeOperatorEmailAddress = config.get('nodeOperatorEmailAddress')
-
   if (!trustedNotifierEnabled && !nodeOperatorEmailAddress) {
     exitWithError(
       'Cannot startup without a trustedNotifierID or nodeOperatorEmailAddress'
     )
   }
 
+  await connectToDBAndRunMigrations()
+
+  // Clear all redis locks
+  try {
+    await redisClient.WalletWriteLock.clearWriteLocks()
+  } catch (e: any) {
+    logger.warn(`Could not clear write locks. Skipping..: ${e.message}`)
+  }
+
+  // This is called `cpus()` but it actually returns the # of logical cores, which is possibly higher than # of physical cores if there's hyperthreading
+  const logicalCores = cpus().length
+  const numWorkers = config.get('expressAppConcurrency') || logicalCores
+  logger.info(`Spawning ${numWorkers} processes to run the Express app...`)
+  _.times(numWorkers, () => cluster.fork())
+
+  cluster.on('exit', (worker, code, signal) => {
+    logger.info(`Worker process with pid=${worker.process.pid} died`)
+  })
+}
```

Comment:
if primary process doesn't do anything further, should it just gracefully exit at the end of this function?

Reply:
actually nvm it needs to stay up to log+re-spawn workers